src/Spryker/Zed/AppPayment/Dependency/Plugin/AppPaymentPlatformPaymentMethodsPluginInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Zed\AppPayment\Dependency\Plugin;
+
+use Generated\Shared\Transfer\PaymentMethodConfigurationRequestTransfer;
+use Generated\Shared\Transfer\PaymentMethodConfigurationResponseTransfer;
+
+interface AppPaymentPlatformPaymentMethodsPluginInterface extends AppPaymentPlatformPluginInterface
+{
+    /**
+     * Specification:
+     * - Receives a `PaymentMethodConfigurationRequestTransfer` with:
+     *   - `PaymentMethodConfigurationRequestTransfer::appConfig`
+     * - Returns a `PaymentMethodConfigurationRequestTransfer`.
+     * - Each PaymentMethod which is listed in PaymentMethodConfigurationRequestTransfer::paymentMethodsToAdd will trigger a `AddPaymentMethod` message.
+     * - Each PaymentMethod which is listed in PaymentMethodConfigurationRequestTransfer::paymentMethodsToDelete will trigger a `DeletePaymentMethod` message.
+     *
+     * @api
+     */
+    public function configurePaymentMethods(
+        PaymentMethodConfigurationRequestTransfer $paymentMethodConfigurationRequestTransfer
```

Comment:
I only want to reduce the things people need to do outside. When this is already part of the request transfer and it needs to be used you can just use it.

Reply:
That is clear, but adding everything developers might need to each object is also not good practice
[Note: Changes are not required]

Reply:
I went through it again to update the docs and the AppConfig is meant to be used when a PSP has several payment methods to be selected. Through the AppConfig you can make the decision inside the implementation. For Stripe it's not used as we only have one PaymentMethod.

And I think in such a package which is meant to be "just" used you should give to the real implementation as much as possible and since we have the object already at hand in this package passing it costs less than if someone would need to get it from the database again.